ui/pages/confirmations/components/signature-request-original/signature-request-original.component.js

```diff
@@ -16,29 +16,26 @@ import {
 } from '../../../../helpers/utils/util';
 import { isSuspiciousResponse } from '../../../../../shared/modules/security-provider.utils';
 import SiteOrigin from '../../../../components/ui/site-origin';
-import Typography from '../../../../components/ui/typography/typography';
 import { PageContainerFooter } from '../../../../components/ui/page-container';
 import {
-  TypographyVariant,
-  FontWeight,
   TextAlign,
   TextColor,
   Size,
+  TextVariant,
   ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
   IconColor,
   Display,
   BlockSize,
-  TextVariant,
   BackgroundColor,
   ///: END:ONLY_INCLUDE_IF
 } from '../../../../helpers/constants/design-system';
 import {
   ButtonLink,
+  Text,
   ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
   Box,
   Icon,
   IconName,
```

Comment:
We should be respecting environment tags `///: BEGIN:ONLY_INCLUDE_IF(build-mmi)` and order of imports. Place the Text component import back inside of the `///: BEGIN:ONLY_INCLUDE_IF(build-mmi)` tag
```suggestion
  ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
  Box,
  Icon,
  IconName,
  Text,
```